serverless/serverless-release-notes.adoc

```diff
@@ -23,6 +23,7 @@ include::modules/serverless-deprecated-removed-features.adoc[leveloffset=+1]
 
 // Release notes included, most to least recent
 // OCP + OSD
+include::modules/serverless-rn-1-22-0.adoc[leveloffset=+1]
```

Comment:
@rh-max The "KafkaBinding API" is removed with Serverless 1.22 so we should include a note about that in the "Deprecated and removed features" page here: https://docs.openshift.com/container-platform/4.10/serverless/serverless-release-notes.html#serverless-tech-preview-features_serverless-release-notes
This feature was deprecated for a few releases and now it's being removed.

Reply:
@mgencur Done, and great point about reviewing the feature matrix tables. If you spot anything else to add to any of the tables ("[Generally Available and Technology Preview features](https://deploy-preview-45026--osdocs.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-tech-preview-features_serverless-release-notes) or [Deprecated and removed features](https://deploy-preview-45026--osdocs.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-deprecated-removed-features_serverless-release-notes)), please let me know

Reply:
Looks good, thanks!